lib/src/base-styles/_nx-btn.scss

```diff
@@ -11,107 +11,110 @@
   @include semi-bold();
   @include container-horizontal;
 
-  background-color: $nx-grey-1;
-  border: $nx-border-dark;
-  border-radius: 4px;
-  color: $nx-text-color-light;
+  background-color: $nx-white;
+  border: 1px solid #045dd1;
+  border-radius: 6px;
+  color: #045dd1;
   cursor: pointer;
   display: inline-block;
-  font-size: 14px;
-  height: 32px;
+  font-size: 16px;
+  height: 40px;
   line-height: 16px;
-  margin: 0 8px;
-  padding: 0 12px;
+  margin: 0 $nx-spacing-xs;
+  padding: 0 $nx-spacing-l;
   text-align: center;
   vertical-align: middle;
   min-width: 100px;
 
-  + .nx-btn {
-    // buttons should be 8px from adjacent items, except other buttons which they should only be 4px from
-    margin-left: -4px;
+  &:hover {
+    background-color: #c5deff;
   }
 
-  &:hover, &:focus, &:active {
-    border-color: $nx-slate;
-    color: $nx-text-color;
+  &:focus {
+    border: 1px solid #43cbf5;
+    outline: 0;
   }
 
   &:active {
-    background-color: $nx-grey-2;
+    background-color: #e8f2ff;
   }
 
   &.disabled, &[disabled] {
-    background-color: $nx-grey-1;
-    border-color: $nx-grey-2;
-    color: $nx-grey-4;
+    background-color: #e9e9e9;
+    border-color: #949494;
+    color: #949494;
     cursor: default;
-
-    &:focus {
-      outline: 0;
-    }
   }
 }
 
 .nx-btn--primary {
-  background-color: $nx-background-color-interactive;
-  border-color: transparent;
-  border-width: 2px;
-  color: $nx-off-white;
+  background-color: #045dd1;
+  border: 1px solid #045dd1;
+  color: $nx-white;
   padding: 0 11px;
 
-  &:hover, &:focus, &:active {
-    border-color: $nx-background-color-interactive-active;
-    color: $nx-off-white;
+  &:hover {
+    background-color: #034396;
   }
 
   &:active {
-    background: $nx-background-color-interactive-active;
-    border-color: $nx-background-color-interactive-active;
+    background-color:  #2783fb;
   }
 
   &.disabled, &[disabled] {
-    background-color: $nx-background-color-interactive-disabled;
-    border-color: transparent;
-    color: $nx-off-white;
+    background-color: #e9e9e9;
+    border-color: #949494;
+    color: #949494;
+    cursor: default;
   }
 }
 
 .nx-btn--tertiary {
-  background-color: transparent;
-  border-color: transparent;
+  border-color: #d1d6f0;
+  color: #2c2c2c;
 
-  &:hover, &:focus {
-    border-color: $nx-grey-4;
+  &:hover {
+    background-color: $nx-white;
+    border-color: #2c2c2c;
   }
 
   &:active {
-    background-color: transparent;
-    border-color: $nx-slate;
+    background-color: #f1f3fa;
+    border-color: #2c2c2c;
   }
 
   &.disabled, &[disabled] {
-    background-color: transparent;
-    border-color: transparent;
+    background-color: #e9e9e9;
+    border-color: #949494;
+    color: #949494;
+    cursor: default;
```

Comment:
The disabled styles are repeated throughout this file.  That repetition could be decreased by using a mixin or possibly by just rearranging the selectors.

Reply:
> The disabled styles are repeated throughout this file. That repetition could be decreased by using a mixin or possibly by just rearranging the selectors.

I put them into a placeholder class which I @extend. However this messed with the cascade so I had to add a bit more specificity: https://github.com/sonatype/sonatype-react-shared-components/pull/39/commits/8fbf3a0cb44202c1c53a9eaa0882a5f8f797a4e8